.github/workflows/deploy_whole_doc.yml

```diff
@@ -69,15 +79,18 @@ jobs:
   deploy_child_repo:
     name: Deploy child repo ${{ matrix.repo.repo_name }}-${{ matrix.repo.doc_version }}
     needs: [prepare-matrix, deploy_framework]
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Same here

Reply:
latest is not really a proper choice because we never now if it will work when the latest ubuntu change on GitHub, it is a better thing to fix the ubuntu version